### ISSUE TITLE
An unknown error occurred - commerce product

### ISSUE THREAD

**Issue by skoften** [venveo/craft-bulkedit#7]
Updated to the latest version so we can edit Commerce Products, but I am getting an unknown error in het CP after selecting some products and clicking on bulk edit. 

Stacktrace:

```
2019-07-22 10:29:44 [-][1][-][error][yii\base\UnknownPropertyException] yii\base\UnknownPropertyException: Getting unknown property: venveo\bulkedit\Plugin::bulkEdit in /var/www/site/craft/vendor/yiisoft/yii2/base/Component.php:154
Stack trace:
#0 /var/www/site/craft/vendor/yiisoft/yii2/di/ServiceLocator.php(77): yii\base\Component->__get('bulkEdit')
#1 /var/www/site/craft/vendor/venveo/craft-bulkedit/src/controllers/BulkEditController.php(53): yii\di\ServiceLocator->__get('bulkEdit')
#2 [internal function]: venveo\bulkedit\controllers\BulkEditController->actionGetFields()
#3 /var/www/site/craft/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#4 /var/www/site/craft/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#5 /var/www/site/craft/vendor/craftcms/cms/src/web/Controller.php(187): yii\base\Controller->runAction('get-fields', Array)
#6 /var/www/site/craft/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('get-fields', Array)
#7 /var/www/site/craft/vendor/craftcms/cms/src/web/Application.php(299): yii\base\Module->runAction('venveo-bulk-edi...', Array)
#8 /var/www/site/craft/vendor/craftcms/cms/src/web/Application.php(566): craft\web\Application->runAction('venveo-bulk-edi...', Array)
#9 /var/www/site/craft/vendor/craftcms/cms/src/web/Application.php(278): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#10 /var/www/site/craft/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#11 /var/www/site/craft/web/index.php(21): yii\base\Application->run()
#12 {main}
2019-07-22 10:29:44 [-][1][-][info][application] $_GET = [
    'p' => 'cp/actions//venveo-bulk-edit/bulk-edit/get-fields'
]

$_POST = [
    'elementIds' => [
        0 => '10261'
        1 => '9581'
    ]
    'requestId' => '1'
    'elementType' => 'craft\\commerce\\elements\\Product'
]

$_FILES = []

$_COOKIE = [
    'CRAFT_CSRF_TOKEN' => '‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢'
    '1031b8c41dfff97a311a7ac99863bdc5_username' => '8597fcb1c537290b66bb33afabd354d85655181b73ac682af8be80205a65e1c7a:2:{i:0;s:41:\"1031b8c41dfff97a311a7ac99863bdc5_username\";i:1;s:4:\"arif\";}'
    '_ga' => 'GA1.2.791794327.1563531932'
    '_gid' => 'GA1.2.1743822713.1563531932'
    '_fbp' => 'fb.1.1563531934084.375698754'
    'CraftSessionId' => 'hf0abl4uu08kqsf2p7b0sk8sc2'
    '1031b8c41dfff97a311a7ac99863bdc5_identity' => '71784daa2e83c9930f18c10c15cf0412c404d2bdcff56142b2b7d2422f3ec4b0a:2:{i:0;s:41:\"1031b8c41dfff97a311a7ac99863bdc5_identity\";i:1;s:254:\"[\"1\",\"[\\\"cEBm3bTqC8QMOfG5PnG-6mTzRdqCHJJpzDJHqz8xN325ZNZog523BiDoBIS4W7AyGn39ookhpHPZuKQUBQgVrzZ3tw6peiHEsfpz\\\",null,\\\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36\\\"]\",25920000]\";}'
    '__stripe_mid' => '9b34fc2d-bb7f-4da2-9698-53590147f2bc'
    '_gat_gtag_UA_750851_59' => '1'
]

$_SESSION = [
    'bd62416aa8538ede709019a5e113eea5__flash' => []
    '1031b8c41dfff97a311a7ac99863bdc5__token' => '‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢‚Ä¢'
    '1031b8c41dfff97a311a7ac99863bdc5__id' => '1'
    '1031b8c41dfff97a311a7ac99863bdc5__expire' => 1589704184
    'enableDebugToolbarForCp' => true
    'commerce_customer' => '1'
    'commerce_cart' => '99a2b9eefb39e9b5eb0fdac69b1bdba8'
    'rv-recent-ids' => [
        0 => 9581
        1 => 9183
        4 => 9317
        5 => 8730
        10 => 10261
        11 => 6408
        15 => 6111
        17 => 8764
    ]
    'bd62416aa8538ede709019a5e113eea5__auth_access' => [
        0 => 'saveAssetInVolume:3'
        1 => 'saveAssetInVolume:155'
        2 => 'saveAssetInVolume:157'
        3 => 'saveAssetInVolume:72'
        4 => 'saveAssetInVolume:69'
        5 => 'saveAssetInVolume:98'
    ]
]
```

Craft CMS 3.2.4.1
Craft Commerce Pro 2.1.10

**Comment by Mosnar** [comment]
Thanks for reporting. I just tried it again myself and everything worked as expected - I'll investigate further today!

**Comment by Mosnar** [comment]
Sorry about that, just released 1.1.0.1 that should fix it! I moved a service declaration out of Composer and it had been cached on my development machine.
